contents/blog/posthog-vs-hotjar.mdx

```diff
@@ -10,626 +10,227 @@ featuredImage: >-
   <ComparisonRow column1={true} column2={true} feature="Data anonymization" description="Remove personally identifiable information." />
   <ComparisonRow column1={true} column2={false} feature="Cookieless tracking option" description="Use session replays and analytics without cookie consent banners" />
 </ComparisonTable>
 
+## Price comparison
+
+> **Note:** Prices correct as of Aug 21, 2024.
+
+PostHog is cheaper than Hotjar for both session replay and user surveys, regardless of whether you choose Hotjar's most popular Business tier, or the more expensive Scale tier that includes Hotjar's basic analytics features, and a dedicated account manager.
+
+### Session replay pricing
+
+Hotjar charges based on 'sessions per day', whereas PostHog uses monthly recorded sessions. We've multiplied Hotjar's daily sessions by 30 to get a monthly equivalent. 
+
+[GRAPH WILL GO HERE]
+
+| Monthly sessions | Hotjar Businss | Hotjar Scale | PostHog |
+|------------------|--------------|-----------------|---------|
+| **5k**               | $99          | $213            | $0      |
+| **15k**              | $99          | $213            | $50     |
+| **45k**              | $249         | $525            | $157    |
+| **75k**              | $389         | $827            | $224    |
+| **120k**             | $489         | Contact sales   | $313    |
+| **240k**             | $789         | Contact sales   | $524    |
+
+At 75k recorded sessions per month, PostHog is $1,980 cheaper over 12 months compared to Hotjar's Business tier, and $7,236 cheaper than Hotjar's Scale tier, when paying monthly.
+
+PostHog also offers a more generous free tier that includes 5,000 replay sessions, 1 million analytics events, and 250 survey responses for free _every month_. Hotjar doesn't offer any free usage outside of its extremely limited Basic tier.
+
+### Surveys pricing
+
+It's similar story for user surveys, where PostHog is cheaper than both Hotjar's Business and Scale tiers. At 2,500 survey responses each month, PostHog is $324 per year cheaper than Hotjar's Business tier when paying monthly.
+
+[GRAPH WILL GO HERE]
+
+| Monthly survey responses | Hotjar Business | Hotjar Scale | PostHog |
+|--------------------------|--------------|-----------------|---------|
+| **250**                      | $79          | $159            | $0      |
+| **500**                      | $79          | $159            | $51     |
+| **1,000**                    | $99          | $259            | $99     |
+| **2,500**                    | $179         | $479            | $152    |
+| **5,000**                    | $289         | Contact sales   | $240    |
+| **7,500**                    | $399         | Contact sales   | $330    |
+| **10,000**                   | $479         | Contact sales   | $418    |
+
+You can see how much PostHog will cost you using the calculator on our [pricing page](/pricing).
+
 ## Frequently asked questions
```

Comment:
I don't mind including questions like:

1. Who is Hotjar useful for?

2. Who is PostHog useful for?

Others:

- Is Hotjar really more focused on the nitty-gritty?

> But Hotjar is much more focused on the nitty-gritty of why users behave the way they do on your website or products. PostHog is more focused on high-level numbers.

- Why do some many people use Hotjar? Why is Hotjar so popular?

- Can product managers, designers, researchers, and marketers use PostHog?